src/Freya.Core/Operators.fs

```diff
@@ -17,46 +17,79 @@
 // limitations under the License.
 //----------------------------------------------------------------------------
 
-module Freya.Core.Operators
+namespace Freya.Core
```

Comment:
I've avoided namespaces everywhere else purely for the sake of consistency. I wonder if we could just split this over two files and leave them as top level modules? The general breakdown and moving of asyncM, returnM, etc. under Freya though I completely agree, however we structure the files - it's a good move, thanks!